app/components/Form/ApplicationForm.tsx

```diff
@@ -364,112 +342,13 @@ const ApplicationForm: React.FC<Props> = ({
     return formData;
   };
 
-  const isCustomPage = customPages.includes(sectionName);
-  const isSubmitPage = pageNumber >= subschemaArray.length;
-  const isAcknowledgementPage = sectionName === 'acknowledgements';
-
-  const submitBtns = (
-    <SubmitButtons
-      disabled={!isSubmitEnabled || isSubmitting}
-      isUpdating={isUpdating}
-      isSubmitPage={isSubmitPage}
-      formData={formData}
```

Comment:
note that `formData={formData}` has been updated below to `formData={formData[sectionName]}`